src/lwc/utilCommon/utilCommon.js

```diff
@@ -528,6 +528,15 @@ const stripNamespace = (apiName , namespacePrefix) => {
     return apiNameParts[1];
 }
 
+/*******************************************************************************
+ * @description Parse proxy objects for debugging, mutating, etc
+ *
+ * @param {object} obj: Object to be parsed
+ */
+const parse = (obj) => {
```

Comment:
We have an existing method called `mutable` that does the same thing here. Could we align to using one or the other? I have no preference regarding the name fwiw